### PR TITLE
databox standalone

### PR DESCRIPTION
**Pull request by calebjclark**
- added @ulixee/databox as a standalone tool
- added @ulixee/databox-for-puppeteer

Neither of these have documentation right now. Wanted to use them in Stream testing; I'm still open to the question of whether they stay around.

### CODE REVIEW

File: databox/client/lib/DataboxWrapper.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import IComponents, { IRunFn } from '../interfaces/IComponents';
+import DataboxInternal from './DataboxInternal';
+import IBasicInput from '@ulixee/databox-interfaces/IBasicInput';
+import IDataboxWrapper from '@ulixee/databox-interfaces/IDataboxWrapper';
+import { setupAutorunBeforeExitHook, attemptAutorun } from './utils/Autorun';
+import IDataboxRunOptions from '@ulixee/databox-interfaces/IDataboxRunOptions';
+
+export default class DataboxWrapper<TInput = IBasicInput, TOutput = any> implements IDataboxWrapper {
+  public static defaultExport: DataboxWrapper;
+  public static disableAutorun = !!process.env.ULX_DATABOX_DISABLE_AUTORUN;
```
Comment (blakebyrnes) [inline databox/client/lib/DataboxWrapper.ts:11]: Should this variable toggle be moved into autorun? You need to explicitly turn off all types of DataboxWrappers if it's setup like this.

File: databox/for-hero/client/index.mjs [hunk 2]
```diff
@@ -1,15 +1,10 @@
+import { setupAutorunMjsHack } from '@ulixee/databox/index.mjs';
```
Comment (blakebyrnes) [inline databox/for-hero/client/index.mjs:1]: Will this correctly delegate to a submodule?

**Comment by blakebyrnes** [comment]
I like the thought exercise here. I don't like the idea that puppeteer is a dependency of the monorepo. Nor the idea of keeping puppeteer dependencies up to date. It also seems like this is adding a module in a half-state to the repo. You would in practice want to be adding fakebrowser or stealth onto pupp, I think. Plus it's missing the core side.

File: databox/client/lib/DataboxWrapper.ts (hunk 1 above)
Comment (calebjclark) [reply databox/client/lib/DataboxWrapper.ts]: Good point. Just pushed a commit that moves the env var check into the constructors.

**Comment by calebjclark** [comment]
I don't think databox-for-puppeteer should add strealth. If someone wants to create databox-for-strealth they can.

**Pull request merged**